documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -864,6 +864,63 @@ The following example shows a typical transaction event message:
+Certain strategies are more reliable than others for specific corner cases around schema changes; however, there are performance and overhead implications that should be considered.
+
+==== Writing the data dictionary to redo logs
+
+The default strategy is called `redo_log_catalog`, and this works by flushing a copy of the data dictionary to the redo logs immediately upon each redo log switch.
+This is the most reliable strategy for tracking schema changes interwoven with data changes because Oracle LogMiner has a way to interpolate between the starting and ending data dictionary states across a series of change vectors.
+
+However, this mode is also the most experience because it requires several key steps to function.
+First, the data dictionary must be flushed to the redo logs on each log switch.
+This can quickly consume valuable space for the database's archive log destination, while also generating substantially more archive logs than the database administrators may have previously accounted for.
+It's imperative that administrators be aware if you intend to use this mode to make sure that the database is configured appropriately.
+
+[IMPORTANT]
+====
+When using the `redo_log_catalog` mode, multiple {prodname} Oracle connectors should not be used to capture changes from the same logical database.
+====
+
+==== Using the online catalog directly
+
+The next strategy mode is called `online_catalog`, and this works vastly different because the data dictionary is never flushed to the redo logs.
+Instead, Oracle LogMiner will use the most current data dictionary state when doing comparisons.
+This has the benefit that it has the least overhead, and operates the fastest; however, at the cost that interweaving schema changes with data changes can sometimes result in an event failure.
+
+Users who see the redo SQL refer to a table by a name like `OBJ# 123456` (where the number is its object identifier) or the columns as `COL1` or `COL2`; this means that a schema change was not performed in lock-step, and Oracle LogMiner was unable to reconstruct the SQL reliably for that change.
+When using this strategy, we recommend that the table's schema and its indices be as static and free from change as possible.
+If you need to perform a schema change using the `online_catalog` mode, it's possible but you must follow these steps:
+
+1. Wait for all existing data changes (DML) to be captured by the connector.
+2. Perform the schema (DDL) change, waiting for the change to be captured by the connector.
+3. Resume data changes (DML) on the table.
+
+This lock-step procedure will guarantee that Oracle LogMiner will be able to safely reconstruct the SQL for all data changes.
+
+==== Hybrid approach
+
+This is a new, experimental strategy that can be enabled by setting the strategy to `hybrid`.
```

Comment:
```suggestion
The `hybrid` log mining strategy is a new, experimental strategy.
```
Should this be conditionalized for community use? (and/or flagged as DP/TP for downstream?)

Reply:
It should :+1:

Reply:
Conditionalized.